acceptance/cluster/docker.go

```diff
@@ -358,7 +358,10 @@ func (cli retryingDockerClient) retry(ctx context.Context, timeout time.Duration
 	for i := 0; i < cli.attempts; i++ {
 		timeoutCtx, _ := context.WithTimeout(ctx, timeout)
 		err := f(timeoutCtx)
-		if err != context.DeadlineExceeded {
+		// docker-engine/client wraps the context.DeadlineExceeded with its own
+		// error message, forcing us to detect deadline exceeded by string
+		// matching.
+		if !strings.Contains(err.Error(), context.DeadlineExceeded.Error()) {
```

Comment:
@tamird I might be able to look at `timeoutCtx.Err()`, but let's just stick with this for now as I know the "deadline exceeded" messages is being output.